internal/core/src/storage/MinioChunkManager.cpp

```diff
@@ -257,8 +262,8 @@ MinioChunkManager::MinioChunkManager(const StorageConfig& storage_config)
 }
 
 MinioChunkManager::~MinioChunkManager() {
-    ShutdownSDKAPI();
     client_.reset();
+    ShutdownSDKAPI();
```

Comment:
why shutdown sdk after reseting client?

Reply:
SDK API is high level than client, so need to close SDKAPI before client at the end.